sdk/cognitiveservices/cognitiveservices-luis-authoring/README.md

```diff
@@ -15,38 +15,45 @@ npm install @azure/cognitiveservices-luis-authoring
 
 ### How to use
 
-#### nodejs - Authentication, client creation and listPhraseLists features as an example written in TypeScript.
+#### nodejs - Authentication, client creation and listApplicationVersionPatternFeatures features as an example written in TypeScript.
 
-##### Install @azure/ms-rest-nodeauth
+##### Install @azure/ms-rest-azure-js
 
 ```bash
-npm install @azure/ms-rest-nodeauth
+npm install @azure/ms-rest-azure-js
 ```
 
 ##### Sample code
 
 ```typescript
-import * as msRest from "@azure/ms-rest-js";
-import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { LUISAuthoringClient, LUISAuthoringModels, LUISAuthoringMappers } from "@azure/cognitiveservices-luis-authoring";
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new LUISAuthoringClient(creds, subscriptionId);
-  const appId = ec7b1657-199d-4d8a-bbb2-89a11a42e02a;
-  const versionId = "testversionId";
-  const skip = 1;
-  const take = 1;
-  client.features.listPhraseLists(appId, versionId, skip, take).then((result) => {
+import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";
+import { LUISAuthoringClient } from "@azure/cognitiveservices-luis-authoring";
+
+import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";
+import { LUISRuntimeClient } from "@azure/cognitiveservices-luis-runtime";
+
+let authoringKey = "<luis-authoring-key>";
+const creds = new CognitiveServicesCredentials(subscriptionId);
+
+// change "westus" in the endpoint url based on your region, check the following link
+// https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-reference-regions
+const client = new LUISAuthoringClient(creds, "https://westus.api.cognitive.microsoft.com/");
```

Comment:
I would suggest we do not hardcode the endpoint in the README. We have seen a lot of first-use developers just copy and paste the snippets without checking the comments and this can lead to a confusing failure.